dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -365,6 +365,16 @@
       <ThemeResource>true</ThemeResource>
       <Link>Themes\%(Filename)%(Extension)</Link>
     </Page>
+    <Page Include="$(MSBuildProjectDirectory)\Density\Compact.xaml">
+      <SubType>Designer</SubType>
+      <ThemeResource>true</ThemeResource>
+      <Link>Density\%(Filename)%(Extension)</Link>
+    </Page>
+    <Page Include="$(MSBuildProjectDirectory)\Density\CompactDatePickerTimePickerFlyout.xaml">
```

Comment:
I think this dictionary should be in a dev\DensityStyles directory or something like that.

Reply:
Renamed